osgp/platform/osgp-domain-core/src/main/java/org/opensmartgridplatform/domain/core/valueobjects/smartmetering/EventType.java

```diff
@@ -76,31 +97,31 @@
     NEW_M_BUS_DEVICE_DISCOVERED_CHANNEL_4(135),
     PERMANENT_ERROR_FROM_M_BUS_DEVICE_CHANNEL_4(136);
 
-    private int value;
+    private static final EventType[] VALUES = EventType.values();
+    private static final Map<Integer, EventType> lookup = new HashMap<>();
```

Comment:
Je kunt het static block vervangen door een oneliner met een stream:
```suggestion
        private static final Map<Integer, EventType> lookup = EnumSet.allOf(EventType.class).stream()
            .collect(Collectors.toMap(EventType::getValue, Function.identity()));
```

Reply:
yes, ook hier weer bestaande code die ik heb verplaatst vanwege een jenkins melding over de volgorde van declaratie.